source/validate_instruction.cpp

```diff
@@ -51,11 +324,252 @@ spv_result_t InstructionPass(ValidationState_t& _,
+      case SpvOpReturn: break;
+      case SpvOpReturnValue: break;
+      case SpvOpUnreachable: break;
+      case SpvOpLifetimeStart: break;
+      case SpvOpLifetimeStop: break;
+      case SpvOpAtomicLoad: break;
+      case SpvOpAtomicStore: break;
+      case SpvOpAtomicExchange: break;
+      case SpvOpAtomicCompareExchange: break;
+      case SpvOpAtomicCompareExchangeWeak: break;
+      case SpvOpAtomicIIncrement: break;
+      case SpvOpAtomicIDecrement: break;
+      case SpvOpAtomicIAdd: break;
+      case SpvOpAtomicISub: break;
+      case SpvOpAtomicSMin: break;
+      case SpvOpAtomicUMin: break;
+      case SpvOpAtomicSMax: break;
+      case SpvOpAtomicUMax: break;
+      case SpvOpAtomicAnd: break;
+      case SpvOpAtomicOr: break;
+      case SpvOpAtomicXor: break;
+      case SpvOpAtomicFlagTestAndSet: break;
+      case SpvOpAtomicFlagClear: break;
+      case SpvOpEmitVertex: break;
+      case SpvOpEndPrimitive: break;
+      case SpvOpEmitStreamVertex: break;
+      case SpvOpEndStreamPrimitive: break;
+      case SpvOpControlBarrier: break;
+      case SpvOpMemoryBarrier: break;
+      case SpvOpGroupAsyncCopy: break;
+      case SpvOpGroupWaitEvents: break;
+      case SpvOpGroupAll: break;
+      case SpvOpGroupAny: break;
+      case SpvOpGroupBroadcast: break;
+      case SpvOpGroupIAdd: break;
+      case SpvOpGroupFAdd: break;
+      case SpvOpGroupFMin: break;
+      case SpvOpGroupUMin: break;
+      case SpvOpGroupSMin: break;
+      case SpvOpGroupFMax: break;
+      case SpvOpGroupUMax: break;
+      case SpvOpGroupSMax: break;
+      case SpvOpEnqueueMarker: break;
+      case SpvOpEnqueueKernel: break;
+      case SpvOpGetKernelNDrangeSubGroupCount: break;
+      case SpvOpGetKernelNDrangeMaxSubGroupSize: break;
+      case SpvOpGetKernelWorkGroupSize: break;
+      case SpvOpGetKernelPreferredWorkGroupSizeMultiple: break;
+      case SpvOpRetainEvent: break;
+      case SpvOpReleaseEvent: break;
+      case SpvOpCreateUserEvent: break;
+      case SpvOpIsValidEvent: break;
+      case SpvOpSetUserEventStatus: break;
+      case SpvOpCaptureEventProfilingInfo: break;
+      case SpvOpGetDefaultQueue: break;
+      case SpvOpBuildNDRange: break;
+      case SpvOpReadPipe: break;
+      case SpvOpWritePipe: break;
+      case SpvOpReservedReadPipe: break;
+      case SpvOpReservedWritePipe: break;
+      case SpvOpReserveReadPipePackets: break;
+      case SpvOpReserveWritePipePackets: break;
+      case SpvOpCommitReadPipe: break;
+      case SpvOpCommitWritePipe: break;
+      case SpvOpIsValidReserveId: break;
+      case SpvOpGetNumPipePackets: break;
+      case SpvOpGetMaxPipePackets: break;
+      case SpvOpGroupReserveReadPipePackets: break;
+      case SpvOpGroupReserveWritePipePackets: break;
+      case SpvOpGroupCommitReadPipe: break;
+      case SpvOpGroupCommitWritePipe: break;
       default:
-        break;
+        return _.diag(SPV_ERROR_INTERNAL) << "NOT IMPLEMENTED";
```

Comment:
Quite apart from the trivial-case explosion above, validation should not fail because the validator is deficient.  That makes it impossible to use the validator in test suites and live drivers.